ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -408,10 +336,30 @@ message DamlCommandDedupKey {
   string application_id = 2;
   string command_id = 3;
 }
+
 message DamlCommandDedupValue {
-  // NOTE(JM): Currently no content. Could store pointer to log entry.
+  google.protobuf.Timestamp record_time = 1;
```

Comment:
Could you add a comment that this is meant to be used in the future for pruning?

Reply:
Done